File: web/src/model/victron_controller/dashboard/from_0_1_0_ws-log-line.ts
// @ts-nocheck
import {WsLogLine as v0_1_0_WsLogLine} from './v0_1_0/WsLogLine'
import {WsLogLine as dashboard_WsLogLine} from './WsLogLine'

export function convert__ws_log_line__from__0_1_0(from: v0_1_0_WsLogLine): dashboard_WsLogLine {
    return new dashboard_WsLogLine (
        from.at_epoch_ms,
        from.level,
        from.source,
        from.message
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_ws_log_line.rs


pub fn convert__ws_log_line__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::ws_log_line::WsLogLine) -> crate::victron_controller::dashboard::ws_log_line::WsLogLine {
    crate::victron_controller::dashboard::ws_log_line::WsLogLine {
        at_epoch_ms: from.at_epoch_ms.clone(),
        level: from.level.clone(),
        source: from.source.clone(),
        message: from.message.clone(),
    }
}